requirements.txt

```diff
@@ -1,5 +1,6 @@
 apischema
 black
+ghapi
```

Comment:
Is this a candidate for inclusion in pcds-envs?

Reply:
I'd argue it's venv-able. I'm starting to be very reluctant to add packages to pcds conda haha. I also think I don't expect everyone to be making gh api calls frequently